$MirOS: ports/comms/linphone/patches/patch-mediastreamer_sndcard_c,v 1.1.7.1 2005/03/18 15:43:50 tg Exp $
--- mediastreamer/sndcard.c.orig	Fri Oct  1 23:24:15 2004
+++ mediastreamer/sndcard.c	Fri Oct  1 23:51:01 2004
@@ -139,6 +139,14 @@ gint sys_audio_manager_init(SndCardManag
 
 #endif
 
+#if defined(__NetBSD__) || defined(__OpenBSD__)
+gint bsd_audio_manager_init(SndCardManager *manager, gint index)
+{
+	manager->cards[index] = oss_card_new("/dev/audio", "/dev/mixer");
+	return 1;
+}
+#endif
+
 #include "osscard.h"
 #include "alsacard.h"
 
@@ -147,6 +155,10 @@ void snd_card_manager_init(SndCardManage
 	gint index=0;
 	gint tmp=0;
 	memset(manager,0,sizeof(SndCardManager));
+	#if defined(__NetBSD__) || defined(__OpenBSD__)
+	tmp=bsd_audio_manager_init(manager,index);
+	index+=tmp;
+	#endif
 	#ifdef HAVE_SYS_SOUNDCARD_H
 	tmp=oss_card_manager_init(manager,index);
 	index+=tmp;
